apps/docker-compose.dist.yml

```diff
@@ -181,7 +181,7 @@ x-brandwatch-api-configuration: &brandwatch-api-configuration
 # "deploy/replicas", we define every shard as its own independent service.
 #
 x-solr-shard_base:   &solr-shard_base
```

Comment:
I remembered why using Docker Hub with with Docker Swarm was painful - even if one does `docker login` to log in to the authenticated (and $50 lighter) account belonging to an organization, Docker daemon [doesn't bother passing those credentials to the Hub](https://docs.docker.com/docker-hub/download-rate-limit/#docker-swarm) unless [one passes the `--with-registry-auth` argument with `docker stack deploy`](https://docs.docker.com/engine/reference/commandline/stack_deploy/).

So, could you add this argument to the "deploy this thing by running this" comment at the top of this file and elsewhere (if appropriate)?

Reply:
well-spotted!